addons/l10n_be/demo/account_demo.py

```diff
@@ -10,12 +10,22 @@ def _get_demo_data(self, company=False):
         if company.account_fiscal_country_id.code == 'BE':
             cid = company.id
             account_data = demo_data.setdefault('account.account', {})
-            account_data.update({
-                f"account.{cid}_a100": {'tag_ids': [Command.link(self.env.ref('account.demo_capital_account').id)]},
-                f"account.{cid}_a300": {'tag_ids': [Command.link(self.env.ref('account.demo_stock_account').id)]},
-                f"account.{cid}_a7600": {'tag_ids': [Command.link(self.env.ref('account.demo_sale_of_land_account').id)]},
-                f"account.{cid}_a6201": {'tag_ids': [Command.link(self.env.ref('account.demo_ceo_wages_account').id)]},
-                f"account.{cid}_a242": {'tag_ids': [Command.link(self.env.ref('account.demo_office_furniture_account').id)]},
-            })
+            tags = {
+                'account.demo_capital_account': 'Demo Capital Account',
+                'account.demo_stock_account': 'Demo Stock Account',
+                'account.demo_sale_of_land_account': 'Demo Sale of Land Account',
+                'account.demo_ceo_wages_account': 'Demo CEO Wages Account',
+                'account.demo_office_furniture_account': 'Office Furniture',
+            }
+            tag_ids = []
+            for ref, tag_name in tags.items():
+                tag = self.env.ref(ref, raise_if_not_found=False) or self.env['account.account.tag'].create({'name': tag_name})
+                tag_ids.append(tag.id)
+
+            codes = ['a100', 'a300', 'a7600', 'a6201', 'a242']
+            for i, code in enumerate(codes):
+                account_data.update({
+                    f"account.{cid}_{code}": {'tag_ids': [Command.link(tag_ids[i])]}
+                })
```

Comment:
I really don't understand why you're creating the demo tag if we don't have the demo data...
Please don't do that.

 ```suggestion
            tag_by_codes = {
                'a100': 'account.demo_capital_account',
                'a300': 'account.demo_stock_account',
                'a7600': 'account.demo_sale_of_land_account',
                'a6201': 'account.demo_ceo_wages_account',
                'a242': 'account.demo_office_furniture_account',
            }
            for code, tag_xml_id in tags.items():
                tag = self.env.ref(tag_xml_id, raise_if_not_found=False)
                account_data.update({
                    f'account.{cid}_{code}': {'tag_ids': [Command.link(tag.id)]},
                })
```
You can also use a dict comprehension if you prefer :)